src/chi-vue/src/views/DataTableView.vue

```diff
@@ -164,17 +169,21 @@ import ColumnCustomization from '../components/column-customization/ColumnCustom
             name: 'input',
             placeholder: 'Input filter',
             type: 'input',
+            id: 'myoptions2',
             value: '',
+            placeholder: 'Your Place Holder',
```

Comment:
```suggestion
```
Please, remove this property or change the value of existing one as you are repeating the `placeholder` property in this object